app/assets/javascripts/validate.js

```diff
@@ -40,6 +40,18 @@ document.addEventListener('turbolinks:load', function() {
               }
             }
             break;
+          case 'phone':
+            if (!value || $.trim(value).length < 1) {
+              notify(this, 'Missing Information');
+            }
+            else if(value){
+              var phoneReg = /^[\+]?[0-9]{0,3}[-\s\.]?[(]?[0-9]{1,3}[)]?[-\s\.]?[0-9]{1,3}[-\s\.]?[0-9]{4,6}$/;
+              if (!phoneReg.test(value)) {
+                notify(this, 'Please enter a valid phone number');
```

Comment:
It looks like this is never being hit, upon testing, the front-end submits all the data but the questionnaire model is kicking it back because a phone number is not being specified. So I don't think we're properly testing it on the front-end.